inst/generate_examples.R

```diff
@@ -1,12 +1,5 @@
 source("tests/testthat/setup.R")
 set.seed(1)
```

Comment:
Do we need to set seed here?

Reply:
this was just leaving what you had before. Aiming for minimal changes. As you said before I think eventually this whole script can go

Reply:
The `set.seed` does nothing (it's just before saving some files) I think so suggest removing

Reply:
```suggestion
```